docs/dev.rst

```diff
@@ -313,3 +313,17 @@ Finally, exit the interactive Docker shell and try to rebuild:
 .. code-block:: shell
 
     $ docker-compose build buildhub
+
+Debugging metrics logging
+=========================
+
+By default, all metrics logging goes to
+``markus.backends.datadog.DatadogMetrics``. This requires that you have a
+``statsd`` server running on ``$STATSD_HOST:STATSD_PORT``. If you don't
+have that locally you can change it to plain Python logging by setting
+``LOG_METRICS=true``. For example:
```

Comment:
Maybe it would be more explicit to have a string value, like `LOG_METRICS=python` or `LOG_METRICS=datadog` (default)

Reply:
Actually, I think that's great idea. I'll switch to that.

Reply:
What should happen if someone accidentally sets `LOG_METRICS=logggging`? I.e. a typo.

Reply:
Yes, it's fine, even a `ValueError`